std_msgs/String mesh_name
std_msgs/String mesh_file
sensor_msgs/Image mask
std_msgs/Int32 mask_offset_x
std_msgs/Int32 mask_offset_y
geometry_msgs/Pose pose